src/main/java/com/zontik/groshiky/exception/MissionTransactionException.java

```diff
@@ -0,0 +1,24 @@
+package com.zontik.groshiky.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class MissionTransactionException extends RuntimeException{
```

Comment:
The same .... probably you meant MissedTransactionException
but it could be NotFoundTransactionException as well, if you like it